<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Message_Window_UI</class>
 <widget class="QMainWindow" name="Message_Window_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>消息</string>
  </property>
  <widget class="QWidget" name="Central_Widget">
   <layout class="QVBoxLayout" name="Central_Widget_Layout">
    <item>
     <widget class="QWidget" name="Massage_Panel" native="true">
      <layout class="QVBoxLayout" name="Massage_Panel_Layout">
       <item>
        <layout class="QHBoxLayout" name="Close_Button_Layout">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Close_Button">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="All_Res.qrc">
             <normaloff>:/all_images/res/Exit.png</normaloff>:/all_images/res/Exit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Massage_Layout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="Massage_Label">
           <property name="text">
            <string>       </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Confirm_Layout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Confirm_Button">
           <property name="text">
            <string>OKK</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="All_Res.qrc"/>
 </resources>
 <connections/>
</ui>
